app/src/main/java/gov/hhs/cdc/trustedintermediary/external/slf4j/DeployedLogger.java

```diff
@@ -0,0 +1,83 @@
+package gov.hhs.cdc.trustedintermediary.external.slf4j;
+
+import gov.hhs.cdc.trustedintermediary.wrappers.Logger;
+import java.util.Arrays;
+import org.slf4j.LoggerFactory;
+import org.slf4j.event.Level;
+
+/**
+ * Humble object interface for logging. Uses SLF4J behind the scenes. The deployed logger doesn't
+ * colorize its messages and uses a logger name that uses the JSON structured logging.
+ */
+public class DeployedLogger implements Logger {
+
+    private static final DeployedLogger INSTANCE = new DeployedLogger();
+    private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger("tilogger");
+
+    private DeployedLogger() {}
+
+    public static DeployedLogger getInstance() {
+        return INSTANCE;
+    }
+
+    @Override
+    public void logTrace(String traceMessage) {
+        Level level = Level.TRACE;
```

Comment:
Why do we need the level variable here and other places where it's called once? Is it for consistency?

Reply:
Uh, good point.  We don't need a separate variable.  I can pass it in directly.  Let me do that.